src/render/src/App.svelte

```diff
@@ -106,8 +157,10 @@ async function getFunctionByLine(
 function setBackgroundColor(colors: "light" | "dark") {
```

Comment:
Probably better to rename to `applyColors` or similar, now that we're not only setting the background color!